pages/docs/environments/gitpod.mdx

```diff
@@ -2,10 +2,35 @@ import DocsLayout from "@components/layouts/DocsLayout.js";
 
 ## Gitpod
 
-In the cloud free development environment which will have all the dependencies you need (for example MongoDB).
+Gitpod is an open-source application that provides a prebuilt development environments in your browser - powered by VS Code.<br />
+It automates the provisioning of a ready-to-code developer environment, so you will have all the dependencies you need (for example MongoDB).<br />
+With Gitpod, there’s no need to download any code to your machine. It can spin up a server-side-dev-environment for you in seconds from any Git Repository.<br />
```

Comment:
```suggestion
With Gitpod, there’s no need to download any code to your machine. It can create a server-side-dev-environment for you in seconds from any repository on GitHub.<br />
```